test/extensions/INTEL/SPV_INTEL_task_sequence/negative/invalid_cluster_mode.ll

```diff
@@ -0,0 +1,59 @@
+; RUN: llvm-as %s -o %t.bc
```

Comment:
Would it make sense to mention in comments source C++ code if any was used to produce ll tests, just for reference purpose, so that it's easier to understand test logic?

Reply:
Not sure if it's needed for the negative tests which are the small reduced versions of the `task_sequence.ll` test. I guess I can add the C++ example to the original test though.

Reply:
Great, thank you